### PR TITLE
[Hibernate]LPS-132873

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test:sf

**Comment by tinatian** [comment]
@vicnate5 Is the ci:test:sf failure real?

Thanks
Tina.

File: nbproject/project.properties [hunk 1]
```diff
@@ -144,6 +144,7 @@ javac.classpath=\
     lib/portal/httpmime.jar:\
     lib/portal/ical4j.jar:\
     lib/portal/im4java.jar:\
+	lib/portal/istack-commons-runtime.jar:\
```
Comment (vicnate5) [inline nbproject/project.properties:147]: @tinatian This is probably the SF issue

Comment (tinatian) [reply nbproject/project.properties:147]: Thanks @vicnate5!

**Comment by tinatian** [comment]
Fix SF and See https://github.com/shuyangzhou/liferay-portal/pull/10092

**Pull request closed without merging**